stock_barcodes/static/src/js/basic_controller.js

```diff
@@ -283,13 +283,8 @@ odoo.define("stock_barcodes.BasicController", function (require) {
                             const $action_confirm = self.$(
                                 "button[name='action_confirm']:visible"
                             );
-                            const $action_confirm_force = self.$(
```

Comment:
Why are you removing these features?

Reply:
In the context of doing a picking. 
The function that was called from these actions, forced the creation a product movement when you scanned a wrong product.
As I explain in the PR, I think is better not doing nothing if the product scaned is not in the picking list.

Reply:
I don't think so. We may need to add extra products in the picking. That's an intended feature.

Reply:
OK, you're right 
Now it works fine. 
Thanks for the comments